magicctapipe/scripts/lst1_magic/lst1_magic_dl2_to_dl3.py

```diff
@@ -607,18 +147,27 @@ def dl2_to_dl3(input_file_dl2, input_dir_irf, output_dir, config):
         **extra_header,
     )
 
-    hdus.append(hdu_aeff)
+    hdus.append(aeff_hdu)
 
+    # Interpolate the energy dispersion with a custom way, since there
+    # is a bug in the function of pyirf v0.6.0 about the renormalization
     logger.info("Interpolating the energy dispersion...")
 
-    edisp_interp = interpolate_energy_dispersion(
```

Comment:
any problem with the pyirf function that required rewriting it here?

Reply:
Yes, please see the issue #94. I opened a pull request #95 to fix this issue separately, but it is already merged to this branch. If the pull request is merged to the master, this will not be the changes done by this pull request.

Reply:
This issue has just now been fixed in the master branch, so let me resolve it.